src/asmcnc/comms/serial_connection.py

```diff
@@ -1020,6 +1030,13 @@ def process_grbl_push(self, message):
                         self.digital_spindle_kill_time = int(digital_spindle_feedback[2])
                         self.digital_spindle_mains_voltage = int(digital_spindle_feedback[3])
 
+                        if self.digital_spindle_ld_qdA > 0:
+                            if self.yp:
+                                if self.spindle_data_error_buffer == 3:
+                                    self.yp.digital_spindle_mains_voltage = self.digital_spindle_mains_voltage
+                                else:
+                                    self.spindle_data_error_buffer += 1
+
```

Comment:
Isn't this the bit that needs removing for now?

Reply:
different thing